Docker/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
```

Comment:
I don't know it's will be working , because this is only a Shell script not  a Bash Script and `set -e` is for the bash as far as I know.